pages/clients/index.tsx

```diff
@@ -59,38 +83,47 @@ function Clients() {
     </>
   );
 
-  const codeBodyTemplate = (rowData: ClientTicket) => (
+  const codeBodyTemplate = (rowData: ServiceRequestType) => (
     <>
       <span className="p-column-title">Code</span>
-      {rowData.ticketNo}
+      {rowData.id}
     </>
   );
 
-  const typeBodyTemplate = (rowData: ClientTicket) => (
+  const typeBodyTemplate = (rowData: ServiceRequestType) => (
     <>
       <span className="p-column-title">Type</span>
-      {rowData.type}
+      FIXME rowData.type
     </>
   );
 
-  const nameBodyTemplate = (rowData: ClientTicket) => (
+  const descriptionBodyTemplate = (rowData: ServiceRequestType) => (
     <>
-      <span className="p-column-title">Name</span>
-      {rowData.name}
+      <span className="p-column-title">Description</span>
+      {rowData.description}
     </>
   );
 
-  const statusBodyTemplate = (rowData: ClientTicket) => (
+  const emailBodyTemplate = (rowData: ServiceRequestType) => (
+    <>
+      <span className="p-column-title">Description</span>
+      FIXME rowData.team_member.email
```

Comment:
There are a few FIXMEs here. Are we ok merging this with those?

Reply:
For now I say yes. I'm trying to keep this PR small, so its just focused on the "Input Form" and not on these other pages. They were previously referring to fields that don't exist. So rather than fix them or totally delete this page, I we can leave it as is until we come back to it in future sprints.

Reply:
Correction, I've moved the input form changes into a different PR.